.circleci/config.yml

```diff
@@ -0,0 +1,48 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:lts
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            # Find a cache corresponding to this specific package-lock.json checksum
+            # when this file is changed, this key will fail
+            - v1-npm-deps-{{ checksum "package-lock.json" }}
+            # Find the most recently generated cache used from any branch
+            - v1-npm-deps-
+      - run:
+          name: Install Dependencies
+          command: npm ci
+
+      - run:
+          name: Install JUnit coverage reporter
+          command: npm install --save-dev jest-junit
+
+      - save_cache:
+          key: v1-npm-deps-{{ checksum "package-lock.json" }}
+          paths:
+            - ~/.npm
+
+      - run:
+          name: Run test
+          command: npm run test
+
+      - run:
```

Comment:
can't we do store_test_results on the folder where junit.xml are?

Reply:
the `junit.xml` is always generated at root dir by default. I didn't managed to change the default output directory of this file. That's why I have one more step to move it to reports (should be moved to dist instead) dir for store_test_results step.